src/main/java/Pages/ProductPage.java

```diff
@@ -61,6 +61,10 @@ public String orderConfirmText() {
         return ExplicitStrategy.waitForElements(productsFromCatalog);
     }
 
+    public List<WebElement> findAllProductsFromSearchAndReturnWebElement() {
```

Comment:
I'm not sure if this method really needed. Seems like can be replaced with getter?